pkg/scheduler/scheduler.go

```diff
@@ -148,7 +150,26 @@ func NewScheduler(cfg Config, limits Limits, log log.Logger, registerer promethe
 		Name: "cortex_query_scheduler_enqueue_duration_seconds",
 		Help: "Time spent by requests waiting to join the queue or be rejected.",
 	})
-	s.requestQueue = queue.NewRequestQueue(s.log, cfg.MaxOutstandingPerTenant, cfg.AdditionalQueryQueueDimensionsEnabled, cfg.QuerierForgetDelay, s.queueLength, s.discardedRequests, enqueueDuration)
+	querierInflightRequestsGauge := promauto.With(registerer).NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "cortex_query_scheduler_querier_inflight_requests",
+			Help: "Number of inflight requests being processed on a querier-scheduler connection.",
```

Comment:
[nit] This is clearer to me:

```suggestion
			Help: "Number of inflight requests being processed on all querier-scheduler connections.",
```